Plots/Scatter/NCL_scatter_6.py

```diff
@@ -0,0 +1,72 @@
+"""
+NCL_scatter_6.py
+===============
+This script illustrates the following concepts:
+   - Drawing a scatter plot with markers of different colors and sizes
+   - Drawing outlined and filled markers on a polar map plot
+   - Generating dummy data using "random"
+   - Changing the marker colors on a polar map plot
+   - Changing the marker sizes on a polar map plot
+   - Turning off y-axis labels
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_6.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_6_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Create dummy data:
+
+numpoints = 100
+lon = np.random.randint(0, 360, numpoints)
```

Comment:
`randint` is a totally valid way of generating random data. I did notice that the NCL script uses a random uniform function. If you'd like a more uniform distribution, there is a [function](https://het.as.utexas.edu/HET/Software/Numpy/reference/generated/numpy.random.uniform.html) in `numpy` for that. It's up to you which you'd like to use.